src/sections/OurDoctors/HeaderTitle.jsx

```diff
@@ -0,0 +1,14 @@
+import HeaderText from "../../components/HeaderText";
+
+const HeaderTitle = () => {
```

Comment:
This is not required, directly use `HeaderText` where required.

Reply:
Solved it. thank you! `HeaderText `is being used directly now!